mdal/frmts/mdal_xmdf.hpp

```diff
@@ -29,7 +29,7 @@ namespace MDAL
    * are stored in single
```

Comment:
also add to docs the XMDF is used by TUFLOW  AND Hydro-as. also write what is the difference between two (tuflow has active flag and single root)

Reply:
you may even use some of this https://github.com/lutraconsulting/MDAL/issues/313#issuecomment-718993039